src/main/java/org/openrewrite/java/logging/ParameterizedLogging.java

```diff
@@ -30,6 +30,7 @@
 import org.openrewrite.java.tree.J;
 import org.openrewrite.java.tree.JavaType;
 import org.openrewrite.java.tree.TypeUtils;
+import org.openrewrite.staticanalysis.groovy.GroovyFileChecker;
```

Comment:
Would it make sense to move these file checkers to the rewrite repo?

Reply:
Kotlin is not in core , so that'll not immediately work I think